### DISCUSSION TITLE
How to get better with MTEX

### DISCUSSION THREAD

**Discussion by sarialzerey** [mtex-toolbox/mtex#2042] [Ask Anything] [answered]
Hi,

I don't have much experience with programming/MATLAB but I read some of the documentation on MTEX's website. I hardly can do what I want and sometime the people on this forum help me to do what I couldn't on my own.
Is it my limited experience with programming or do I need to learn more about texture analysis?

Please guide me how to get better with this. If possible, please recommend Textbooks for texture analysis and advise me how to be better with MTEX coding.

Thanks!

**Comment by zmichels** [comment, marked as answer]
Hi,

There is an annual MTEX Workshop that is always enlightening and covers a range of background and applications details.

https://mtex-toolbox.github.io/workshop24

It's a great question about what's the best approach. I don't have a great answer or suggestion. I sort of learned MTEX and Matlab parallel to my learning about texture and quantitative microstructural analysis. I found, as you mentioned, the MTEX tutorials and forum are very helpful.

**Comment by Savillian** [comment]
I was in the same boat as you starting out - trial by fire in someways, but systematically working your way through how one command works in an example and then the next will pay off. It's hard at the beginning, but building that foundation has helped me greatly. Commenting what each coding step is doing was also a big boost to my understanding AND retention of what my operations were doing.

As for texture - I recommend Texture and Anisotropy by Kocks, Tome, and Wenk and Introduction to Texture Analysis by Engler and Randle.

There are several great presentations from schools, national labs, and companies on texture as well - happy to direct you to some if those are of interest. I've also seen some papers which summarize texture well in an applied context (e.g., additive manufacturing and solidification) which could be useful.

**Reply by sarialzerey** [reply]
Thank you for the comprehensive reply. I didn't know the Anisotropy textbook, I will check it out.
and yes please, I need papers that describe in-plane orientations and disorientation in general as I feel I'm week in the two topics. thanks in advance!

**Reply by Savillian** [reply]
My pleasure! Here's a paper which discusses texture, sampling statistics, phase transformations, and processing: [https://doi.org/10.1016/j.addma.2022.103023](https://doi.org/10.1016/j.addma.2022.103023). A few in the author group have other investigations in texture you can reference.

I don't have the files with me, but Tony Rollett at Carnegie-Mellon has great documentation online for understanding texture.
